setup.py

```diff
@@ -13,13 +13,12 @@
     'six>=1.2.0',
 ]
 
-if PY_VERSION == (2, 6):
-    requirements.append('argparse')
-
 if PY_VERSION < (3, 0):
-    long_description = open('README.rst').read() + '\n\n' + open('HISTORY.rst').read()
+    long_description = open('README.rst').read() + '\n\n' + \
```

Comment:
Better to use `(`, `)` for multi-line statements than `\`

Reply:
agreed